pages/cuidado-compartilhado/raps-hospitalar/index.js

```diff
@@ -58,67 +61,147 @@ const RapsHospitalar = ({ }) => {
+                depois={{status:true, descricao:'mas atendidos'}}></CardInternacaoStatus>
+              : <Spinner theme="ColorSM" />
+            }
+          </>,
+          <>
+            { internacoesRapsAltas12m
+              ?<CardPeriodosInternacao
+                periodo={ 'Ano'}
+                descricao={ 'Internações finalizadas entre ' + internacoesRapsAltas12m.a_partir_de_mes + ' de ' + internacoesRapsAltas12m.a_partir_de_ano + ' e ' + internacoesRapsAltas12m.ate_mes + ' de ' + internacoesRapsAltas12m.ate_ano}></CardPeriodosInternacao>
+              : <Spinner theme="ColorSM" />
+            }
+          </>,
+          <>
+            { internacoesRapsAltas12m
+              ?<CardIndicadorDescricao
+                indicador={ internacoesRapsAltas12m.altas_atendimento_raps_antes_nao_apos_nao }
+                descricao={ 'Usuários' }></CardIndicadorDescricao>
+              : <Spinner theme="ColorSM" />
+            }
+          </>,
+          <>
+            { internacoesRapsAltas12m
+              ?<CardIndicadorDescricao
+                indicador={ internacoesRapsAltas12m.altas_atendimento_raps_antes_sim_apos_nao }
+                descricao={ 'Usuários' }></CardIndicadorDescricao>
+              : <Spinner theme="ColorSM" />
+            }
+          </>,
+          <>
+            { internacoesRapsAltas12m
+              ?<CardIndicadorDescricao
+                indicador={ internacoesRapsAltas12m.altas_atendimento_raps_antes_sim_apos_sim }
+                descricao={ 'Usuários' }></CardIndicadorDescricao>
+              : <Spinner theme="ColorSM" />
+            }
+          </>,
+          <>
+            { internacoesRapsAltas12m
+              ?<CardIndicadorDescricao
+                indicador={ internacoesRapsAltas12m.altas_atendimento_raps_antes_nao_apos_sim }
+                descricao={ 'Usuários' }></CardIndicadorDescricao>
+              : <Spinner theme="ColorSM" />
+            }
+          </>,
+          <>
+            { internacoesRapsAltas12m
+              ?<CardPeriodosInternacao
+                periodo={ 'Mês'}
+                descricao={ 'Internações finalizadas no mês selecionado abaixo:' }
+                filtro = {
+                  <FiltroCompetencia
+                    width={'100%'}
+                    dados = {periodosECompetencias}
+                    valor = {filtrosPeriodosInternacoesRapsAltas}
+                    setValor = {setFiltroPeriodoInternacoesRapsAltas}
+                    isMulti
+                  />
+                }
+              ></CardPeriodosInternacao>
+              : <Spinner theme="ColorSM" />
+            }
+          </>,
+          <>
+            { internacoesRapsAltas12m
+              ?<CardIndicadorDescricao
+                indicador={ internacoesRapsAltasFiltradas.reduce((acumulador, valorAtual) => acumulador + valorAtual.altas_atendimento_raps_antes_nao_apos_nao, 0)}
+                indicadorDescricao={ 'Usuários' }></CardIndicadorDescricao>
```

Comment:
Oi @jvns2010! Muda a prop `indicadorDescricao` para `descricao` nos componentes `CardIndicadorDescricao`, por favor?

```suggestion
                descricao={ 'Usuários' }></CardIndicadorDescricao>
```